virtio-devices/src/mem.rs

```diff
@@ -119,6 +119,8 @@ pub enum Error {
     ResizeInval(String),
     // Fail to resize trigger
     ResizeTriggerFail(DeviceError),
+    // Invalid configuration
+    InvalidConfiguration(String),
```

Comment:
Never put a String into and error like this. (I see the code already has that and it's bad.) Create a new `ValidateError` using thiserror/anyhow.

Reply:
Ok I will update :)